src/screens/login.tsx

```diff
@@ -0,0 +1,144 @@
+import { StatusBar } from 'expo-status-bar';
+import {Alert, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {useState} from "react";
+import { NavigationContainer} from "@react-navigation/native";
+import {createReactNativeMatcher} from "@expo/metro-config/build/transformer/createMatcher";
+
+const axios = require('axios');
+
+
+
+export default function LoginScreen({ navigation }: {navigation: any}) {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const loginFailAlert = (err:any) => Alert.alert(
+    "Login Failed",
+    err.response.data,
+    [
+      {
+        text: "Ok",
+        onPress: () => {
+        }
+      }
+    ]
+  )
+  const loginAuth = (uName : string, pWord: string) => {
+    axios({
+      url: "http://localhost:3000/api/user/login",
+      method: "POST",
+      data: {
+        email: uName,
+        password: pWord
+      }
+    }).then((response:string) => {
+      // console.log(response);
+      console.log("successful auth");
```

Comment:
Can we navigate to classroom finder page when there's successful login? (we can navigate to the main search page when it's ready)